CHANGELOG.md

```diff
@@ -0,0 +1,235 @@
+# Changelog
+
+## Unreleased (planned v0.4.0)
+
+### ⚠️ Breaking changes ⚠️
+
+- Core
+  - `isEnabled` configuration option removed #909 
```

Comment:
Should the github issue/PR numbers be changed to URLs?

Reply:
Probably. I'd say these should probably be commit IDs but not sure if that is feasible a depending on how a PR is merged, commit hash might change. May be we can take another pass on before release and update with commit ids.